packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -122,12 +125,17 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
   private container: HTMLDivElement;
   private titleId: string;
   private descriptionId: string;
+  private isModal: boolean;
 
   constructor(props: WizardProps) {
     super(props);
     const newId = Wizard.currentId++;
-    this.titleId = `pf-wizard-title-${newId}`;
-    this.descriptionId = `pf-wizard-description-${newId}`;
+    this.isModal = !props.isInPage;
+    if (this.isModal) {
+      this.titleId = `pf-wizard-title-${newId}`;
+      this.descriptionId = `pf-wizard-description-${newId}`;
+    }
```

Comment:
not sure if this is needed, but the titleId/descriptionId are not used when in-page